install/linux/usr/bin/odemis-mic-selector

```diff
@@ -64,43 +63,70 @@ def count_usb_device(usbid):
     return out.count(usbid)
 
 
+def find_backend_pcus() -> List[model.HwComponent]:
+    """
+    return (list of HwComponent): all the power control units found
+    :raises: IOError if no power control unit is found
+    """
+    pcus = []
+    try:
+        comps = model.getComponents()
+    except (Pyro4.errors.CommunicationError, IOError, LookupError):
+        raise IOError("Failed to access the backend")
```

Comment:
Wouldn't it be nicer to split this up and make a distinction between backend not accessible and backend active but no PCU found?
```suggestion
    except (Pyro4.errors.CommunicationError, IOError):
        raise IOError("Failed to access the backend")
    except LookupError:
        raise LookupError("Failed to find any PCU")    
```

Reply:
All these exceptions actually are a sign that there is no back-end. If there is a backend, but no PCU, that's later... and normally it's fine to return an empty list.
The main issue is the docstring which states that it returns IOError if there are no PCU, actually, it's if there is no backend. -> fixed.